debian/bionic/foreman/rules

```diff
@@ -33,9 +33,10 @@ build:
 	dh $@ --with=systemd
 
 override_dh_installinit:
-	dh_installinit
+	dh_installinit --only-scripts
```

Comment:
I did this to stop foreman.service from being created during `apt-get install foreman`, however, I wondered more broadly if this should be dropped all together in favor of systemd?

Reply:
Well, technically, Debian still supports non-systemd systems, so if that package would go into Debian proper, that'd be a bug.

Given we don't aim for that, we could say we only support systemd systems.

Reply:
FWIW, `dh_installinit` is still responsible for also *copying* the `foreman.service` files etc, so this line can't be dropped completely yet. But at some point we will be able to. (Needs to bump some other places to make `dh_systemd_*` take over the copying)